frontend/controller/backend.js

```diff
@@ -14,14 +14,40 @@ const languageFileMap = new Map([
 ])
 
 sbp('okTurtles.events/on', NOTIFICATION_TYPE.VERSION_INFO, (versionInfo) => {
+  if (versionInfo.GI_VERSION === process.env.GI_VERSION) {
+    // No refresh necessary, we're already at the latest version
+    sessionStorage.removeItem(NOTIFICATION_TYPE.VERSION_INFO)
+    return
+  }
+
   console.info('New Group Income version available:', versionInfo)
-  // Prevent the client from trying to reconnect when the page starts unloading.
-  sbp('okTurtles.data/get', PUBSUB_INSTANCE).destroy()
   // TODO: allow the user to manually reload the page later.
+  try {
+    // Store the current VERSION_INFO in session storage to prevent infinite
+    // reload loops
+    const existingSerialized = sessionStorage.getItem(NOTIFICATION_TYPE.VERSION_INFO)
+    if (existingSerialized) {
+      const existingVersionInfo = JSON.parse(existingSerialized)
+      if (
+        Array.isArray(existingVersionInfo) &&
+        !(Date.now() - existingVersionInfo[0] >= 1e7) &&
```

Comment:
What is this 1e7 value? 2.8 hours? Why not use `HOURS_MILLIS` instead?

Reply:
Yes, that's approximately 2.8 hours. It sounded like a reasonable value.

Reply:
Could you use `HOURS_MILLIS` for readability?